packages/js/src/workouts/components/WorkoutCard.js

```diff
@@ -27,7 +27,11 @@ export default function WorkoutCard( {
 	upsellText,
 	workout,
 	badges,
+	blocked,
 } ) {
+	// Only make configuration workout a prerequisite for other workouts for users that can do the configuration workout.
+	blocked = blocked && window.wpseoWorkoutsData.canDoConfigurationWorkout;
```

Comment:
I'd move this "upwards" to `OrphanedWorkoutCard.js` and `CornerstoneWorkoutCards` so the `WorkoutCard` itself remains agnostic about that

Reply:
Agreed!